### ISSUE TITLE
Adding a plug-in to the Google Chrome plug-in store will cause the browser to crash.

### ISSUE THREAD

**Issue by gugulu18** [kiwibrowser/src.next#644]
**Describe the bug**
Adding a plug-in to the kiwi browser through the Google plug-in store will cause the browser to crash, and the plug-in cannot be added normally.
I translated my language through google translate, the sentence in question is hopefully understandable, thank you.
Chromium | 103.0.5060.102 (正式版本) （64 位）
操作系统 | Android 10; SEA-AL10 Build/HUAWEISEA-AL10
The system is HarmonyOS
2.0.0.234 (C00E230R6P6)（Android 10）
The phone is Huawei nova5pro

**To Reproduce**
After clicking +from store through the browser extension, after entering the chrome online store, click any plug-in, click Add to Chrome, the browser crashes, and the plug-in cannot be added normally.

**Expected behavior**
Hope to get back to normal use.

**Smartphone (please complete the following information):**
 - Device: HUAWEI nova5pro
 - Android Version : A10
 - Kiwi Browser Version :103.0.5060.102

**Comment by gugulu18** [comment]
Also, clicking Close All Tabs requires another click to close all tabs.

**Comment by gnadelwartz** [comment]
Thus is a known bug of 103 dev Version, will be fixed in next dev releases.

**Comment by kiwibrowser** [comment]
It doesn't crash for me.
I'm trying to recreate the problem (and to find a solution).

I will answer back here very soon.

**Comment by Ibuprophen** [comment]
I just tried to install a few plug-ins from the Chrome Store and, after a second (or so) after selecting the blue install button, the button then shows "Checking..." and the browser crashes as reflected on the following image. 

[Kiwi Screenshot 1](https://user-images.githubusercontent.com/18060271/177057615-1373126e-233b-447a-af5c-81820b5d1d3b.jpg)

I tried a ton of options and the same happens. 

I don't actually know which version this may/might have begun since I haven't tried to install any additional add-on for a while. 

I only have 5 installed. 

~Ibuprophen

**Comment by rickninja765** [comment]
> I just tried to install a few plug-ins from the Chrome Store and, after a second (or so) after selecting the blue install button, the button then shows "Checking..." and the browser crashes as reflected on the following image. 
> 
> [Kiwi Screenshot 1](https://user-images.githubusercontent.com/18060271/177057615-1373126e-233b-447a-af5c-81820b5d1d3b.jpg)
> 
> I tried a ton of options and the same happens. 
> 
> I don't actually know which version this may/might have begun since I haven't tried to install any additional add-on for a while. 
> 
> I only have 5 installed. 
> 
> ~Ibuprophen 
> 

Yeah I tried to install new extension but eventually it crashed browser.

**Comment by Ultimatinium** [comment]
@kiwibrowser 
@gugulu18 

Have the similar bug, can't recreate the bug again on the latest release...

Seems like it's fixed...

Android 10
Redmi 6a
armv7
Kiwi : Generation 2610541136 (Fixed Bug)

**Comment by catbabylon** [comment]
Also fixed for me on Samsung S22 Ultra

**Comment by gugulu18** [comment]
@Ultimatinium
Yes, he fixed the problem that installing plug-ins will cause the browser to flash back in the new version.

**Comment by mlwrx1978** [comment]
Not fixed, the problem still exists

**Comment by Ibuprophen** [comment]
I had updated the app yesterday (via this Github Repo). 

I had just tried to install Add-ons, a few minutes ago, and was successful. I even installed a few others and was successful for those. 

In checking out the change(s) pushed for the last update, It didn't look like it had anything specifically to do with the Add-ons (as a personal opinion). 

@kiwibrowser, In my "opinion", I've got a feeling that this recent issue(s), specific to the add-ons, just might be something tied to the Google Chrome Store itself rather than this app. 

~Ibuprophen

**Comment by mlwrx1978** [comment]
It's not the problem of the Google Store. I downloaded the .crx. After installing it locally, the plug-in is not enabled. Click the switch to open the browser and crash. Change the .crx suffix to .zip format.  Normal use
